wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationServiceTests.swift

```diff
@@ -149,6 +149,87 @@ final class ConversationServiceTests: MessagingTestBase {
         XCTAssertEqual(performedAction.isReadReceiptsEnabled, false)
     }
 
+    func test_CreateOneToOneConversation_Team_Success() throws {
+        // Given
+        let team = createTeam(in: uiMOC)
+        user1.teamIdentifier = team.remoteIdentifier
+
+        let oneToOneConversation = createOneToOneConversation(
+            with: user1,
+            in: uiMOC
+        )
+
+        let didFinish = expectation(description: "didFinish")
+
+        // Mock
+        let mockActionHandler = MockActionHandler<CreateGroupConversationAction>(
+            result: .success(oneToOneConversation.objectID),
+            context: uiMOC.notificationContext
+        )
+
+        // When
+        sut.createOneToOneConversation(user: user1) {
+            switch $0 {
+            case .success(let conversation):
+                // Then we got back newly created conversation.
+                XCTAssertEqual(conversation, oneToOneConversation)
+                didFinish.fulfill()
+
+            case .failure(let error):
+                XCTFail("unexpected error: \(error)")
+            }
+        }
+
+        // Then the action was performed with correct arguments.
+        XCTAssert(waitForCustomExpectations(withTimeout: 0.5))
+        let performedAction = try XCTUnwrap(mockActionHandler.performedActions.first)
+
+        XCTAssertEqual(performedAction.messageProtocol, .proteus)
+        XCTAssertEqual(performedAction.qualifiedUserIDs, [user1.qualifiedID].compactMap(\.self))
+        XCTAssertEqual(performedAction.unqualifiedUserIDs, [])
+        XCTAssertEqual(performedAction.name, nil)
+        XCTAssertEqual(performedAction.teamID, team.remoteIdentifier)
+    }
+
+    func test_CreateOneToOneConversation_NoTeam_Success() throws {
```

Comment:
If the method is only for team 1:1 conversations, then we don't actually want this test right?

Reply:
Removed